{"uuid": "f40d920e-66be-4d76-81bd-55f765823bba", "counter": 7, "stdout": "\u001b[1;31mfatal: [192.168.1.1]: UNREACHABLE! => {\"changed\": false, \"msg\": \"Failed to connect to the host via ssh: ssh: connect to host 192.168.1.1 port 22: Connection refused\", \"unreachable\": true}\u001b[0m", "start_line": 5, "end_line": 6, "runner_ident": "5af8c8d9-6b50-4044-a443-396b1ab72810", "event": "runner_on_unreachable", "pid": 5599, "created": "2022-04-22T13:13:19.091178", "parent_uuid": "9e73ed77-7fb2-55f8-e0c6-00000000000f", "event_data": {"playbook": "deploy.yaml", "playbook_uuid": "8aa1a173-4910-4c74-8e2f-be93300c8ee7", "play": "this is a  hello world example", "play_uuid": "9e73ed77-7fb2-55f8-e0c6-000000000007", "play_pattern": "all", "task": "Gathering Facts", "task_uuid": "9e73ed77-7fb2-55f8-e0c6-00000000000f", "task_action": "gather_facts", "task_args": "", "task_path": "/Users/devops/data/project/python/pgoops/upload/iac/helloworld/project/deploy.yaml:2", "host": "192.168.1.1", "remote_addr": "192.168.1.1", "start": "2022-04-22T13:13:19.071478", "end": "2022-04-22T13:13:19.091086", "duration": 0.019608, "res": {"unreachable": true, "msg": "Failed to connect to the host via ssh: ssh: connect to host 192.168.1.1 port 22: Connection refused", "changed": false}, "uuid": "f40d920e-66be-4d76-81bd-55f765823bba"}}